### PR TITLE
l10n: uk: update core translation

### PR DESCRIPTION
**Pull request by arkid15r**

### CODE REVIEW

File: po/uk.po [hunk 1]
```diff
@@ -1138,56 +1138,56 @@ msgstr "неприпустимий %s"
...
 
 #: builtin/index-pack.c:1203
 msgid "pack has junk at the end"
-msgstr ""
+msgstr "мотлох наприкінці пакунка"
 
 #: builtin/index-pack.c:1215
 msgid "confusion beyond insanity in parse_pack_objects()"
-msgstr ""
+msgstr "плутанина на межі пекельного борошна в parse_pack_objects()"
```
Comment (kasya) [inline po/uk.po:1169]: 🤣

Comment (kasya) [inline po/uk.po:1169]:
```suggestion
msgstr "плутанина за межею божевілля в parse_pack_objects()"
```

File: po/uk.po [hunk 2]
```diff
@@ -1138,56 +1138,56 @@ msgstr "неприпустимий %s"
...
 #: builtin/index-pack.c:1249
 #, c-format
 msgid "unable to create thread: %s"
-msgstr ""
+msgstr "не вдалося створити потік: %s"
 
 #: builtin/index-pack.c:1282
 msgid "confusion beyond insanity"
-msgstr ""
+msgstr "плутанина на межі пекельного борошна"
```
Comment (kasya) [inline po/uk.po:1182]:
```suggestion
msgstr "плутанина за межею божевілля"
```

**Review by kasya — COMMENTED** [review]
💪🏼

File: po/uk.po (hunk 1 above)
Comment (arkid15r) [reply po/uk.po:1169]: ☝️

**Pull request merged**